### DISCUSSION TITLE
Simple event duplication

### DISCUSSION THREAD

**Discussion by johnniehard** [vkurko/calendar#238] [Show and tell]
I seeked out this project because I've always been frustrated with how difficult conventional calendar applications make it to duplicate events. In graphics software like Affinity Designer, tldraw, or Adobe Illustrator, it's always really simple, just hold alt/option while dragging and that will duplicate the selected objects.

Turns out this was really straight forward to implement:
```
eventDrop: (info) => {
  const { id, ...oldEvent } = info.oldEvent;

  // Duplicate!
  if (info.jsEvent.altKey) {
    ec.addEvent(oldEvent);
  }
};
```

https://github.com/vkurko/calendar/assets/9264787/4737698d-2719-4f14-b840-127f207de184

**Comment by vkurko** [comment]
Thanks for sharing your code and experience :smiley:
